### PR TITLE
chore(react-scheduler-demos): add integrated data shaping demo

### PR DESCRIPTION
**Pull request by AryamnovEugeniy**
image

### CODE REVIEW

**Comment by AryamnovEugeniy** [comment]
FPR

**Comment by AryamnovEugeniy** [comment]
FPR

**Comment by AryamnovEugeniy** [comment]
FPR

File: packages/dx-react-scheduler-demos/src/demo-sources/scheduler-featured-integrated-data-shaping/material-ui/demo.tsx [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    },
+  },
+  highPriorityAppointment: {
+    borderLeft: `4px solid ${teal[500]}`,
+  },
+  middlePriorityAppointment: {
+    borderLeft: `4px solid ${blue[500]}`,
+  },
+  lowPriorityAppointment: {
+    borderLeft: `4px solid ${indigo[500]}`,
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-demos/src/demo-sources/scheduler-featured-integrated-data-shaping/material-ui/demo.tsx]: To remove this display bug set please`borderBottom: 0`.
image

File: packages/dx-react-scheduler/demos/featured/integrated-data-shaping.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# React Scheduler Integrated Data Shaping
+
+This demo shows how to use TypeScript to create a `Scheduler` with [date navigation](../../docs/guides/date-navigation.md) and [view switching](../../docs/guides/view-switching.md) in the `Uncontrolled mode`. Use this mode if you do not need to share the `Scheduler`’s state among other parts of your application.
```
Comment (RomanTsukanov) [inline packages/dx-react-scheduler/demos/featured/integrated-data-shaping.md]:
```suggestion
This demo shows how to use TypeScript to create a Scheduler with [date navigation](../../docs/guides/date-navigation.md) and [view switching](../../docs/guides/view-switching.md) in Uncontrolled mode. Use this mode if you do not need to share the Scheduler’s state with other parts of your application.
```

**Pull request merged**